src/rosdiscover/models/move_base.py

```diff
@@ -74,7 +78,7 @@ def base_plugin_local(name):
         # type: (str) -> None
         c.pub("~{}/global_plan".format(name), "nav_msgs/Path")
```

Comment:
Prefer `f"{name}/global_plan"

Reply:
Fixed in all that code (not sure I wrote this originally, but it was a simple regexp replace)